stored_queries/ncbi_tax/ncbi_taxon_get_descendants.yaml

```diff
@@ -0,0 +1,21 @@
+# Get the array of descendants for any taxon
+# Defaults to immediate children only
+name: ncbi_taxon_get_descendants
+params:
+  type: object
+  required: [key]
+  properties:
+    key:
+      type: string
+      title: Document key
+      description: Key of the taxon vertex for which you want to find descendants
+    levels:
+      type: integer
+      default: 1
+      title: Number of levels
+      description: How many levels of descendants to traverse and aggregate
+query: |
+  for t in ncbi_taxon
+      filter t._key == @key
+      for c in 1..@levels INBOUND t ncbi_child_of_taxon
+          return c
```

Comment:
Should there be a limit here? This could return a huge amount of data and maybe even oom the machine